javalin/src/test/java/io/javalin/TestMultipartForms.kt

```diff
@@ -267,4 +269,39 @@ class TestMultipartForms {
         //and verify it
         assertThat(response).isEqualTo(expected)
     }
+
+    @Test
+    fun `changing the multipart config correctly sets it`() = TestUtil.test{ app, http ->
+
+        //note: this test does not check the cache directory is set correctly as there is no way to know which
+        //paths exist and are writable on the system the test is being run on.  However, if the other parameters
+        //are read successfully
+        app.updateConfig {
+            it.fileUpload.maxFileSize(100,FileUploadConfig.Units.MB)
```

Comment:
I think `FileUploadConfig.Units` should probably be `SizeUnit` or `ByteUnit`, similar to how Java has `TimeUnit` for time.

Reply:
I've renamed that to ByteUnit as I think that is the clearest explanation of what it is